src/components/Selfie/Intro.js

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react'
+import { h, Component } from 'preact'
+import PageTitle from '../PageTitle'
+import style from './style.css'
+import theme from '../Theme/style.css'
+import classNames from 'classnames'
+import Button from '../Button'
+import { localised } from '../../locales'
+
+class Intro extends Component<Props, State> {
+
+  renderInstruction = (instruction) => {
+    return <li key={instruction.key} className={style.introBullet}>
+      <span className={classNames(style.introIcon, style[`${instruction.key}Icon`])} />
+      <span className={classNames(style.bolder, style.introText)}>{instruction.text}</span>
+    </li>
+  }
+
+  renderInstructions = (instructions) => {
+    return <div className={classNames(style.thinWrapper, style.introCopy)}>
+      <ul className={style.introBullets} aria-label="TODO">
+      {
+        instructions.map(instruction =>
+          this.renderInstruction(instruction)
+        )
+      }
+      </ul>
+    </div>
+  }
+
+  render() {
+    const { translate, nextStep } = this.props
+    return <div className="theme.fullHeightContainer">
+      <PageTitle title={translate("capture.face.intro.title")} subTitle={translate("capture.face.intro.subtitle")} />
+      {this.renderInstructions([{key:"selfie", text:translate("capture.face.intro.selfie_instruction")},{key:"glasses", text:translate("capture.face.intro.glasses_instruction")}])}
```

Comment:
Think this can be made more readable by assigning the instructions into a variable like `instructionsList` and give the render function a more explicit name like `renderInstructionsList` (I almost thought there was duplicate function names in the code further up)

Reply:
Also, looking at the Abstract design (latest) the copy doesn't match up. Would be worth confirming with design about expected copy.

Reply:
You can make this a functional component

Reply:
moved to stateless functional component